tools/integration-tests/README.md

```diff
@@ -0,0 +1,53 @@
+# Integration tests with Docker
+
+![Integration testing setup](../../images/integration-testing-setup.png)
+
+Running the integration tests spins up a GoShimmer network within Docker as schematically shown in the figure above.
+`N` defines the number of `peer_replicas` and can be specified when running the network.
+The peers can communicate freely within the Docker network and this is exactly how the tests are run using the `tester` container.
+Test can be written in regular Go style while the framework provides convenience functions to access a specific peer's web API or logs.
+
+The visualizer, `master_peer's` dashboard and web API are reachable from the host system on the respective ports.
```

Comment:
I think we should mention that the git submodules need to be initialized to have glumb visualizer ready.

Reply:
I was referring to the autopeering network visualizer. I'll make that more clear. But we could of course also expose the glumb visualizer. Though that probably needs some work with the new layer approach?